diff -urN camstream-0.26.2.org/camstream/ftpput.cc camstream-0.26.2/camstream/ftpput.cc
--- camstream-0.26.2.org/camstream/ftpput.cc	2003-08-18 14:09:17.000000000 +0200
+++ camstream-0.26.2/camstream/ftpput.cc	2003-08-18 14:08:32.000000000 +0200
@@ -35,18 +35,17 @@
 
 void CFTPPut::Usage()
 {
-   qDebug("
-Not enough/wrong arguments. Syntax:
-
-   ftpput file [options] [file ...] user:password@server[/path]
-   
-The file is uploaded to the server using the FTP protocol. Path can be
-relative to the user's directory or absolute (use a double /).
-
-Options:
-  --passive         Use passive FTP to upload data
-
-");
+   qDebug("\n"
+"Not enough/wrong arguments. Syntax:\n"
+"\n"
+"   ftpput file [options] [file ...] user:password@server[/path]\n"
+"\n"   
+"The file is uploaded to the server using the FTP protocol. Path can be\n"
+"relative to the user's directory or absolute (use a double /).\n"
+"\n"
+"Options:\n"
+"  --passive         Use passive FTP to upload data\n"
+"\n");
 }
 
 /** 
